iphone/Classes/TiUIWindowProxy.m

```diff
@@ -920,6 +921,45 @@ -(void)setToolbar:(id)items withObject:(id)properties
 	
 }
 
+-(void)setHidesBarsOnSwipe:(id)value
+{
```

Comment:
They come to the proxies as NSNumbers (because of the JavaScriptCore below it). So usually this is what you do in boolean-setters:
```objc
- (void)setHidesBarsOnSwipe:(id)value
{
    ENSURE_TYPE(value, NSNumber);
 
    [myView setWhatever:[TiUtils boolValue:value def:YES]];
    [self replaceValue:value forKey:@"hidesBarsOnSwipe" notification:NO];
}
```